calendar_switzerland/models/calendar_switzerland.py

```diff
@@ -28,3 +28,37 @@ def _send_mail_to_attendees(self, template_xmlid, force_send=False):
     def send_invitation_to_partner(self):
         return super(Attendee, self)._send_mail_to_attendees(
             'calendar.calendar_template_meeting_invitation', True)
+
+
+class CalendarEvent(models.Model):
+    _inherit = "calendar.event"
+
+    campaign_event = fields.Many2one('utm.campaign', 'Campaign')
```

Comment:
The field should be named `campaign_event_id` to follow the naming convention of relational fields.